src/reggae/reggae.d

```diff
@@ -273,8 +273,7 @@ private enum reggaeFileDubSelectionsJson =
         "fileVersion": 1,
         "versions": {
                 "dub": %s,
-                "reggae": {"path":"packages/reggae"},
-                "unit-threaded": %s
+                "reggae": {"path":"packages/reggae"}
         }
```

Comment:
I only put unit-threaded in here because otherwise the selections file wouldn't match and trigger a rebuild. How is it not doing it now?

Reply:
? buildgen doesn't depend on unit-threaded, nor does dub.

Reply:
Right, but at the time I was copying the selections file from reggae itself, and because dub is silly, it included unit-threaded despite it not being used for the configuration in question. 

I guess I just answered my own question.